code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -279,7 +279,7 @@
 	id = "medigax_torso"
 	build_type = MECHFAB
 	build_path = /obj/item/mecha_parts/part/medigax_torso
-	materials = list(/datum/material/iron=20000,/datum/material/glass=10000,/datum/material/diamond=2000)
+	materials = list(/datum/material/iron=20000, /datum/material/glass=10000, /datum/material/gold=2000, /datum/material/silver=10000)
```

Comment:
Look, I understand that diamonds are rare and all that, but that's what makes mechs rare enough. Why change it?

Reply:
Because it's a civvie mech, why would it use diamond for the plating? Infact, why would a military mech use diamond in its armor? The idea is that silver = antibacterial and therefore a good choice.